applications/RomApplication/tests/test_structural_lspg_rom.py

```diff
@@ -0,0 +1,122 @@
+        expected_output_filename = "ExpectedOutputLSPGROM.npy"
+
+        with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
+            # Set up simulation
+            with open(parameters_filename,'r') as parameter_file:
+                parameters = KratosMultiphysics.Parameters(parameter_file.read())
+            model = KratosMultiphysics.Model()
+            self.simulation = rom_testing_utilities.SetUpSimulationInstance(model, parameters)
+
+            # Run test case
+            self.simulation.Run()
+            
+            # Check results
+            expected_output = np.load(expected_output_filename)
+            obtained_output = rom_testing_utilities.GetVectorNodalResults(self.simulation._GetSolver().GetComputingModelPart(), KratosMultiphysics.DISPLACEMENT)
+            aux_nodal_area = rom_testing_utilities.GetNodalAreaVector(self.simulation._GetSolver().GetComputingModelPart())
+            nodal_area = [aux_nodal_area[int((i-1)/2) if i%2 != 0 else int(i/2)] for i in range(2*len(aux_nodal_area))]
+
+            numerator = 0.0
+            denominator = 0.0
+            for i in range(len(obtained_output)):
+                if expected_output[i] > 1.0e-12:
+                    numerator += nodal_area[i]*((1 - (obtained_output[i]/expected_output[i]))**2)
+                    denominator += nodal_area[i]
+            l2 = np.sqrt(numerator/denominator)*100
+            self.assertLess(l2, self.relative_tolerance)
+
+            # Save the obtained results in TEMPERATURE variable and visualize
+            output_results = False
+            if output_results:
+                i = 0
+                for node in self.simulation._GetSolver().GetComputingModelPart().Nodes:
+                    node.SetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_X, 0, obtained_output[i])
+                    i += 1
+                    node.SetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y, 0, obtained_output[i])
+                    i += 1
+                self.PostProcess("test_rbf_shape_functions_utility_1x1_square_G")
+        
+
+    def testStructuralDynamicLSPGRom2D(self):
+        self.work_folder = "structural_dynamic_test_files/LSPGROM/"
+        parameters_filename = "ProjectParametersLSPGROM.json"
+        expected_output_filename = "ExpectedOutputLSPGROM.npy"
+
+        time_snapshots = [2,4,6,8,10]
+
+        with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
+            # Set up and run simulation
+            with open(parameters_filename,'r') as parameter_file:
+                parameters = KratosMultiphysics.Parameters(parameter_file.read())
+            model = KratosMultiphysics.Model()
+            self.simulation = rom_testing_utilities.SetUpSimulationInstance(model, parameters)
+
+            # Patch the RomAnalysis class to save the selected time steps results
+            def Initialize(cls):
+                super(type(self.simulation), cls).Initialize()
```

Comment:
```suggestion
                super().Initialize()
```
No longer needed as we don't support Python 2 anymore.

Reply:
I also noticed this, but due to the way this is being used (to susbtitute a given function inside the class using the types library), it is actually necessary to do it this way